examples/gnrc_border_router/Makefile

```diff
@@ -76,11 +76,14 @@ QUIET ?= 1
 TAP ?= tap0
 IPV6_PREFIX ?= 2001:db8::/64
 
+# Hook building the host tools and starting ethos into the `make term` target
+TERMDEPS += ethos-start
+
 include $(RIOTBASE)/Makefile.include
 
-.PHONY: host-tools
+.PHONY: ethos-start host-tools
 
-term: host-tools
+ethos-start: host-tools
```

Comment:
this won't work, ethos-start *replaces* the usual terminal program.
Shouldn't this makefile just set TERMPROG and TERMFLAGS to what is now ethos-start?